app/views/unattended/provisioning_templates/iPXE/kickstart_default_ipxe.erb

```diff
@@ -27,8 +27,8 @@ ping --count 1 ${netX/gateway} || echo Ping to Gateway failed or ping command no
 echo Trying to ping DNS: ${netX/dns}
 ping --count 1 ${netX/dns} || echo Ping to DNS failed or ping command not available.
 
-kernel <%= "#{@host.url_for_boot(:kernel)}" %> initrd=initrd.img ks=<%= ks %> <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options", variables: {ipxe_net: true}).strip %>
+kernel <%= "#{@host.url_for_boot(:kernel)}" %> initrd=initrd.img <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options", variables: {ipxe_net: true}).strip %>
 initrd <%= "#{@host.url_for_boot(:initrd)}" %>
 imgstat
 sleep 2
-boot
+boot
```

Comment:
Ah can you revert this one, here is special iPXE handling for ks so please just do the normal `ks` to `inst.ks` change here. Also keep the trailing newline that can cause issues sometimes (not here specifically tho).

Reply:
These should both be fixed in e0197b0.  Thank you for your patience while I stumble through some of this.  It's been a bit of a git learning experience as well.